frontend/src/layout/navigation-3000/sidebars/insights.ts

```diff
@@ -36,11 +36,11 @@ export const insightsSidebarLogic = kea<insightsSidebarLogicType>([
     })),
     reducers(() => ({
         infiniteInsights: [
```

Comment:
3000 sidebar. Rolled out to 0% of users and isn't likely to be rolled out soon. https://us.posthog.com/project/2/feature_flags/21108.